_posts/en/newsletters/2019-10-30-newsletter.md

```diff
@@ -17,10 +18,15 @@ the Bitcoin StackExchange.
 
 ## Action items
 
-- **Help test release candidates:** experienced users are encouraged to
-  help test the latest release candidates for the upcoming versions of
-  [Bitcoin Core][Bitcoin Core 0.19.0] and [C-Lightning][c-lightning
-  0.7.3].  <!-- FIXME: verify latest state Tuesday night -->
+- **Upgrade to C-Lightning 0.7.3:** this latest [release][c-lightning
+  0.7.3] adds support for a PostgresSQL backend, makes it possible to
+  send funds directly to a particular address when closing a channel,
+  and allows you keep your HD wallet seed encrypted when `lightningd`
+  isn't running---plus many other features and several bug fixes.
+
+- **Help Bitcoin Core release candidate:** experienced users are encouraged to
```

Comment:
Should this be 'Help test Bitcoin Core release candidate'?

Reply:
Agree. Perhaps "Help test the Bitcoin Core release candidate:"